tests/integration/ceph_upgrade_test.go

```diff
@@ -157,8 +161,8 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 
 	message := "my simple message"
 	preFilename := "pre-upgrade-file"
-	assert.NoError(s.T(), s.k8sh.WriteToPod("", rbdPodName, preFilename, message))
-	assert.NoError(s.T(), s.k8sh.ReadFromPod("", rbdPodName, preFilename, message))
+	require.NoError(s.T(), s.k8sh.WriteToPod("", rbdPodName, preFilename, message))
```

Comment:
Why change these to `require`? Reading/writing to the pod doesn't really prevent the remainder of the test from executing, right?